target/linux/mediatek/filogic/base-files/etc/hotplug.d/firmware/11-mt76-caldata

```diff
@@ -34,6 +34,17 @@ case "$FIRMWARE" in
 		ln -sf /tmp/tp_data/MT7986_EEPROM.bin \
 			/lib/firmware/$FIRMWARE
 		;;
+	tplink,ex820v)
+		mkdir /tmp/calmnt/
+		ubiattach -m 5 -d 1
```

Comment:
Are these numbers stable?
The ubiattach command has these paramaters:
````
"-p, --dev-path=<path> path to MTD device node to attach\n"
"-m, --mtdn=<number>   MTD device number to attach (alternative method, e.g\n"
"                      if the character device node does not exist)\n"
`````
Is it possible to use `-p` instead of `-m`?

Reply:
did you look into this?